src/napari_deeplabcut/_widgets.py

```diff
@@ -632,3 +684,116 @@ def dropEvent(self, event):
         """
         self._update_property("drag", False)
         self.sig_dropped.emit(event)
+
+
+class ColorSchemeDisplay(QScrollArea):
+
+    class LabelPair(QWidget):
+        def __init__(self, color: str, name: str, parent: QWidget):
+            super().__init__(parent)
+
+            self._color = color
+            self._part_name = name
+
+            self.color_label = QLabel("", parent=self)
+            self.part_label = QLabel(name, parent=self)
+
+            self.color_label.setToolTip(name)
+            self.part_label.setToolTip(name)
+
+            self._format_label(self.color_label, 10, 10)
```

Comment:
Are there ways to avoid hardcoding dimensions here? Would it be better to play with the layout these LabelPairs are added to rather?

Reply:
It's mainly to make them square, do you think their size/shape should vary ?